src/Cake.Common.Tests/Fixtures/Tools/MSBuildRunnerFixture.cs

```diff
@@ -34,8 +34,8 @@ public MSBuildRunnerFixture(bool is64BitOperativeSystem, PlatformFamily platform
                 "/Program86/MSBuild/14.0/Bin/amd64/MSBuild.exe",
                 "/Program86/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/MSBuild.exe",
                 "/Program86/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/amd64/MSBuild.exe",
-                "/Program86/Microsoft Visual Studio/2019/Preview/MSBuild/Current/Bin/MSBuild.exe",
-                "/Program86/Microsoft Visual Studio/2019/Preview/MSBuild/Current/Bin/amd64/MSBuild.exe",
+                "/Program86/Microsoft Visual Studio/2019/Professional/MSBuild/Current/Bin/MSBuild.exe",
```

Comment:
Is there a reason that you chose `Professional` here, rather than Enterprise, similar to what was done for 2017?

Reply:
As I only have the Professional version, I can verify only that path.

Reply:
But if you want, for consistency reasons, I can us the same as it is fairly obvious that it should be correct.

Reply:
There is no "need" for them to be consistent, I was just curious.

Reply:
I think it would be good if we added tests for the other editions as well.

Reply:
@patriksvensson Should I have a look at that (that would probably mean making the `MSBuildRunnerFixture` configurable (to "fake" different VS installation"). Or should that be done probably later as I think cake should have VS2019 support ready asap.